php-mode.el

```diff
@@ -11,7 +11,7 @@
 (defconst php-mode-version-number "1.15.3"
   "PHP Mode version number.")
 
-(defconst php-mode-modified "2015-01-31"
+(defconst php-mode-modified "2015-02-02"
```

Comment:
You modified only test, not `php-mode` itself. Is it necessary to change this value ?